### ISSUE TITLE
Regression in 3.0.1 importing individual bootstrap files

### ISSUE THREAD

**Issue by paulcwatts** [kaliber5/ember-bootstrap#913]
Version 3.0.1 seems to have introduced a regression importing individual bootstrap files. The following works with 3.0.0:

```
@import "ember-bootstrap/functions";
```

In 3.0.1 I get the error:

```
Sass Syntax Error (SassCompiler) in /var/folders/y9/rrlpv2qn7f19ld6mwnclxww80000gn/T/broccoli-64730KDBfhUHEIW2F/out-433-broccoli_merge_trees_full_application//app/styles/cannon/base/_bootstrap.scss:1:9

Error: Can't find stylesheet to import.
  ╷
1 │ @import "ember-bootstrap/functions";
  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
```

Output from `ember bootstrap:info`:

```
Npm packages:
ember-bootstrap: 3.0.1 -> 3.0.1
ember-cli: ~3.12.0 -> 3.12.0
bootstrap: ~4.3.0 -> 4.3.1
bootstrap-sass: n/a
ember-cli-sass: ^10.0.1 -> 10.0.1
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
bootstrapVersion: 4
importBootstrapFont: false
importBootstrapCSS: false
whitelist: undefined
```

FWIW, the whitelist is not undefined, here is my config: 

```
    "ember-bootstrap": {
      bootstrapVersion: 4,
      importBootstrapFont: false,
      importBootstrapCSS: false,
      whitelist: [
        "bs-button",
        "bs-form",
        "bs-modal",
        "bs-nav",
        "bs-navbar",
        "bs-collapse",
        "bs-tooltip"
      ]
    },
```

**Comment by simonihmig** [comment]
Thanks for reporting! I could confirm this, caused by the changes in #910. This only happens when the import is in a nested file, not when it is it the root folder, e.g. `app.scss`. That's why the internal build did not fail here...

I am going to revert #910, as the underlying issue is fixed upstream in https://github.com/embroider-build/embroider/pull/341

**Comment by jelhan** [comment]
@simonihmig Do you think we need test coverage for this one as well as Embroider? Could add tests build with [ember-cli-addon-tests](https://github.com/tomdale/ember-cli-addon-tests). Have gathered some experience with it while working on ember-cli-content-security-policy in the last month. But these kind of tests are quite heavy and will consume a fair amout of our CI time.

**Comment by simonihmig** [comment]
> Could add tests build with ember-cli-addon-tests

I think that's not necessary. We could add a ember-try scenario that adds the `@embroider/*` dependencies, and in (our own) `ember-cli-build.js` we could add the embroider boilerplate in case the dependencies are detected. Created #915 to track.

**Comment by simonihmig** [comment]
Released in 3.0.2!

**Comment by paulcwatts** [comment]
Thanks for the quick response! 💯
